starlette/routing.py

```diff
@@ -340,16 +348,22 @@ def __init__(
         self.path = path.rstrip("/")
         if app is not None:
             self.app: ASGIApp = app
+            routes = getattr(self.app, "routes", [])
```

Comment:
Is this variable used?

Reply:
Ah good catch, it is not being used. It's leftover from a previous implementation.